src/tests/illumidesk/authenticators/test_lti13_authenticator.py

```diff
@@ -117,8 +117,7 @@ async def test_authenticator_returns_username_in_auth_state_with_with_name(monke
     ):
         with patch.object(LTI13LaunchValidator, 'validate_launch_request', return_value=True):
             result = await authenticator.authenticate(request_handler, None)
-            monkeypatch.setitem(auth_state_dict, 'name', 'foo')
-
+            auth_state_dict['name'] = 'Foo'
             assert result['name'] == auth_state_dict['name']
```

Comment:
In this case we know exactly the value that the method should return us so I think that we can change the **assert** step with a fixed value eg. result['name'] == 'foo'

Reply:
Yes, simpler.